packages/cli/src/main/kotlin/elide/tool/engine/EngineTests.kt

```diff
@@ -17,16 +17,29 @@ package elide.tool.engine
 
 import io.micronaut.context.annotation.Bean
 import io.micronaut.context.annotation.Requires
+import io.micronaut.context.condition.Condition
+import io.micronaut.context.condition.ConditionContext
 import org.graalvm.polyglot.Context
 import org.graalvm.polyglot.Engine
 import org.graalvm.polyglot.Source
-import elide.runtime.gvm.jvm.cfg.JvmRuntimeConfig
-import elide.runtime.gvm.python.cfg.PythonRuntimeConfig
-import elide.runtime.gvm.ruby.cfg.RubyRuntimeConfig
 import elide.tool.annotations.EmbeddedTest
 import elide.tool.testing.SelfTest
 import elide.tool.testing.TestContext.Companion.assertDoesNotThrow
 
+
+abstract class LanguageCondition constructor (private val language: String): Condition {
+  override fun matches(context: ConditionContext<*>): Boolean {
+    TODO("Not yet implemented")
```

Comment:
is this intentionally left without implementation?

Reply:
LMFAO no